integration-tests/features/dependency-resolution.feature

```diff
@@ -1,12 +1,5 @@
 Feature: Dependency resolution
 
-  Scenario: launch a stack_group with dependencies
```

Comment:
Do we need to remove this? Whilst the 'stack group' APIs are going away, the concept of a stack group will remain (and certainly the example should still be valid - launching `3` should launch `3/A`, `3/B`, and `3/C`).